narayana/scripts/hudson/jenkins.sh

```diff
@@ -128,6 +149,7 @@ else
   separator=":"
 fi
 
+# run the benchmarks
```

Comment:
Thanks @mmusgrov  for the PR. If I well understand the proper benchmarks is not called here, but on line 118 (https://github.com/jbosstm/performance/blob/master/narayana/scripts/hudson/jenkins.sh#L118 )
I think that in this line only the report generator is invoked and the output is appended to the benchmark output.